docs/pages/billing/usage-based-pricing.mdx

```diff
@@ -0,0 +1,160 @@
+
+- Click **Usage** in the navigation menu.
+- Under the **EAS Update** section, you will find details on updated users and global edge bandwidth details.
+
+<ImageSpotlight
+  alt="Detailed EAS Update usage in the Expo dashboard."
+  src="/static/images/billing/usage-03.png"
+  style={{ maxWidth: 600 }}
+/>
+
+### Enable notifications for EAS Build usage
+
+You can enable **Plan credit usage** notifications to closely monitor your EAS Build usage. It enables email notifications when 80% and 100% of your plan's EAS Build credit is used.
+
+To enable EAS Build credit usage notification:
+
+- Click **Email notifications** in the navigation menu.
+- Under **EAS Build notifications**, click **Subscribe** for **Plan credit usage notifications**.
+
+<ImageSpotlight
+  alt="Enabling EAS Build plan credit usage notifications in the Expo dashboard."
+  src="/static/images/billing/usage-04.png"
+  style={{ maxWidth: 720 }}
+/>
+
+### How to optimize build usage
+
+You can use [EAS Update](/eas-update/introduction/) and [development builds](/develop/development-builds/introduction/) to test and deploy new code without having to create an entirely new build. This will help you iterate faster and reduce build usage.
+
+For most apps, the JavaScript code changes more frequently than the underlying native code and configuration. If you are building a new build every time for code changes, consider [using EAS Update to take advantage of the different iterations](/eas-update/how-it-works/) between JavaScript and native code. This way, you can ship those changes as an update instead.
+
+When using Continuous Integration (CI)/Continuous Deployment (CD) to build pre-production code, you can reduce unnecessary usage by automating the process of building only when changes are made to the native code. You can create a workflow in your CI/CD using [Expo Fingerprint](https://expo.dev/blog/fingerprint-your-native-runtime) to detect when your native code has changed, and only execute a build if it has changed. Otherwise, publish an update if the native code has not changed.
+
+A development build can run any EAS Update that is compatible with its native runtime. If you are using EAS Update with multiple testing channels, you can reduce the need for creating additional builds by having your testers or test devices use the same development build.
+
+### How to optimize update usage
+
+You can manage certain assets to include or exclude when using EAS Update. This reduces the number of assets uploaded or downloaded from the updates server and the global edge bandwidth used.
+
+To optimize the update usage, you can choose to exclude assets that haven't been modified. For example, images or videos that haven't been changed can be excluded. Excluded assets won't be uploaded to the update server and won't be downloaded by the app. However, it's important to make sure that assets that are not part of an update are included in the native build of the app.
```

Comment:
Could add a link to this feature: https://docs.expo.dev/eas-update/asset-selection/